syserol/impute.py

```diff
@@ -16,58 +16,127 @@ def flatten_to_mat(tensor, matrix):
     return tMat
 
 
-def increase_missing(comps, PCAcompare=False):
-    samples = np.array([15, 1000, 5000, 12000, 20000, 28000, 36000, 44000, 52000, 60000, 68000, 76000, 82000, 90000])
+def missingness(c):
+    """ Calculate the missingness (NaN) in an array"""
+    vTop, vBot = 0.0, 0.0
+    for cc in c:
+        vTop += np.sum(np.isfinite(cc))
```

Comment:
Can't this whole thing just be `np.mean(np.isnan(X))`?

Reply:
True, but we also want to handle cases when there are both a tensor and a matrix